src/extras/PMREMGenerator.js

```diff
@@ -0,0 +1,832 @@
+
+		}
+
+	},
+
+	/**
+	 * Disposes of the PMREMGenerator's internal memory. Note that PMREMGenerator is a static class,
+	 * so you should not need more than one PMREMGenerator object. If you do, calling dispose() on
+	 * one of them will cause any others to also become unusable.
+	 */
+	dispose: function () {
+
+		_blurMaterial.dispose();
+
+		if ( _cubemapShader != null ) _cubemapShader.dispose();
+		if ( _equirectShader != null ) _equirectShader.dispose();
+
+		for ( var i = 0; i < _lodPlanes.length; i ++ ) {
+			_lodPlanes[ i ].dispose();
+		}
+
+	},
+
+};
+
+function _createPlanes() {
+
+	var _lodPlanes = [];
+	var _sizeLods = [];
+	var _sigmas = [];
+
+	var lod = LOD_MAX;
+	for ( var i = 0; i < TOTAL_LODS; i ++ ) {
+
+		var sizeLod = Math.pow( 2, lod );
+		_sizeLods.push( sizeLod );
+		var sigma = 1.0 / sizeLod;
+		if ( i > LOD_MAX - LOD_MIN ) {
+
+			sigma = EXTRA_LOD_SIGMA[ i - LOD_MAX + LOD_MIN - 1 ];
+
+		} else if ( i == 0 ) {
+
+			sigma = 0;
+
+		}
+		_sigmas.push( sigma );
+
+		var texelSize = 1.0 / ( sizeLod - 1 );
+		var min = - texelSize / 2;
+		var max = 1 + texelSize / 2;
+		var uv1 = [ min, min, max, min, max, max, min, min, max, max, min, max ];
+
+		var cubeFaces = 6;
+		var vertices = 6;
+		var positionSize = 3;
+		var uvSize = 2;
+		var faceIndexSize = 1;
+
+		var position = new Float32Array( positionSize * vertices * cubeFaces );
+		var uv = new Float32Array( uvSize * vertices * cubeFaces );
+		var faceIndex = new Float32Array( faceIndexSize * vertices * cubeFaces );
+
+		for ( var face = 0; face < cubeFaces; face ++ ) {
+
+			var x = ( face % 3 ) * 2 / 3 - 1;
+			var y = face > 2 ? 0 : - 1;
+			var coordinates = [
+				x, y, 0,
+				x + 2 / 3, y, 0,
+				x + 2 / 3, y + 1, 0,
+				x, y, 0,
+				x + 2 / 3, y + 1, 0,
+				x, y + 1, 0
+			];
+			position.set( coordinates, positionSize * vertices * face );
+			uv.set( uv1, uvSize * vertices * face );
+			var fill = [ face, face, face, face, face, face ];
+			faceIndex.set( fill, faceIndexSize * vertices * face );
+
+		}
+		var planes = new BufferGeometry();
+		planes.setAttribute( 'position', new BufferAttribute( position, positionSize ) );
+		planes.setAttribute( 'uv', new BufferAttribute( uv, uvSize ) );
+		planes.setAttribute( 'faceIndex', new BufferAttribute( faceIndex, faceIndexSize ) );
+		_lodPlanes.push( planes );
+
+		if ( lod > LOD_MIN ) {
+
+			lod --;
+
+		}
+
+	}
+	return { _lodPlanes, _sizeLods, _sigmas };
+
+}
+
+function _allocateTargets( equirectangular ) {
+
+	var params = {
+		magFilter: NearestFilter,
+		minFilter: NearestFilter,
+		generateMipmaps: false,
+		type: equirectangular ? equirectangular.type : UnsignedByteType,
+		format: equirectangular ? equirectangular.format : RGBEFormat,
+		encoding: equirectangular ? equirectangular.encoding : RGBEEncoding,
+		depthBuffer: false,
+		stencilBuffer: false
+	};
+	var cubeUVRenderTarget = _createRenderTarget( params );
+	cubeUVRenderTarget.depthBuffer = equirectangular ? false : true;
```

Comment:
I think this should be equivalent?

This is how it was before:
```js
var cubeUVRenderTarget = _createRenderTarget(
	{ ...params, depthBuffer: ( equirectangular ? false : true ) } );
```

Reply:
Ah, I was afraid if I didn't put that in the constructor that it would have gotten allocated already.

Reply:
Hmm, I'll double check the code, but it seems like it didn't matter 🤔

Reply:
You'd know better than I would, I just wasn't taking the chance.